### PR TITLE
switch postgres template to latest (9.5) version

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
[testextended][extended:core(postgresql)]

File: test/extended/util/db/mysql.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ func (m MySQL) IsReady(oc *util.CLI) (bool, error) {
 		"mysqladmin -h 127.0.0.1 -uroot ping").Output()
 	if err != nil {
 		switch err.(type) {
-		case *exec.ExitError:
+		case *ExitError:
```
Comment (bparees) [inline test/extended/util/db/mysql.go]: @mfojtik you broke the crap out of our extended tests when you changed the return type of .OutPut() in https://github.com/openshift/origin/pull/10130
:(

File: test/extended/util/db/common.go [hunk 2]
```diff
@@ -51,7 +51,7 @@ func executeShellCommand(oc *util.CLI, podName string, command string) (string,
 	out, err := oc.Run("exec").Args(podName, "--", "bash", "-c", command).Output()
 	if err != nil {
 		switch err.(type) {
-		case *exec.ExitError:
+		case *ExitError:
```
Comment (liggitt) [inline test/extended/util/db/common.go]: maybe keep the `*exec.ExitError` case as well for good measure?

**Comment by bparees** [comment]
[merge]

**Pull request merged**